src/datachain/toolkit/ultralytics.py

```diff
@@ -0,0 +1,141 @@
+from typing import Union
```

Comment:
[Q] How are these changes related to video models?

Reply:
I didn't review this file.

Reply:
Moved to separate PR: https://github.com/iterative/datachain/pull/900